### ISSUE TITLE
Install Error when using External drive

### ISSUE THREAD

**Issue by tadghh** [msys2/msys2.github.io#303]
- TODO: Add more info
![image](https://github.com/msys2/msys2.github.io/assets/47073445/1a3c5e7e-9681-4f1f-82bd-7f7517c68662)

**Comment by Biswa96** [comment]
Have you tried to exclude the installation directory in antivirus or antimalware software?

**Comment by tadghh** [comment]
This issue was encountered here too
https://github.com/appveyor/ci/issues/3777

## Workaround
Go=to Windows Security -> App and browser control -> Exploit protection

Need to disable the following
- Force randomization ASLR
- Randomize memory allocations (Bottom-up ALSR)
- Validate exception chains (SEHOP)
- Validate heap intergrity

## Notes
Requireing systems to disable these settings could harm the adoption of Msys2.

**Comment by MehdiChinoune** [comment]
AFAIK ASLR is not enabled by default on Windows.

**Comment by tadghh** [comment]
> AFAIK ASLR is not enabled by default on Windows.

This is correct, however with this "issue" the user base who require those features are being cut off imo.

**Comment by MehdiChinoune** [comment]
> > AFAIK ASLR is not enabled by default on Windows.
> 
> This is correct, however with this "issue" the user base who require those features are being cut off imo.

We can do nothing about it, While MinGW-w64 (native Windows) packages were built with ASLR enabled https://www.msys2.org/news/#2021-01-31-aslr-enabled-by-default. but unfortunately CygWin (Which MSYS environment is based on) has some issues with it.
There were some reports like yours before, like this https://github.com/msys2/MSYS2-packages/issues/3925
see also:
https://cygwin.com/pipermail/cygwin/2020-March/244106.html
https://cygwin.com/pipermail/cygwin/2020-August/245973.html

**Comment by tadghh** [comment]
> > This is correct, however with this "issue" the user base who require those features are being cut off imo.
> 
> We can do nothing about it, While MinGW-w64 (native Windows) packages were built with ASLR enabled https://www.msys2.org/news/#2021-01-31-aslr-enabled-by-default. but unfortunately CygWin (Which MSYS environment is based on) has some issues with it. There were some reports like yours before, like this [msys2/MSYS2-packages#3925](https://github.com/msys2/MSYS2-packages/issues/3925) see also: https://cygwin.com/pipermail/cygwin/2020-March/244106.html https://cygwin.com/pipermail/cygwin/2020-August/245973.html

Hopefully someday it will be fixed :)
